### DISCUSSION TITLE
Improper Input Validation in httpx

### DISCUSSION THREAD

**Discussion by michealroberts** [encode/httpx#2234] [Potential Issue]
I'm seeing the following dependabot alert for this package:

```
Encode OSS httpx <=1.0.0.beta0 is affected by improper input validation in httpx.URL, httpx.Client and some functions using httpx.URL.copy_with.
```

Is this on the roadmap as a fix?

**Comment by nhtgl** [comment]
Hi @michael-observerly 👋  
I've found discussion #2184 which lead to #1831. With v0.23.0 the issue was mitigated and there is an ongoing verification if that's the case. From what I understood, until this is done @dependabot will display this as a potential security issue.
